ask/alexa_io.py

```diff
@@ -13,15 +13,20 @@
 }"""
 
 
+class InvalidAppIdException(Exception):
+    pass
+
+
 class Request(object):
     """
     Simple wrapper around the JSON request
     received by the module
     """
-    def __init__(self, request_dict, metadata=None):
+    def __init__(self, request_dict, metadata=None, supported_app_ids=None):
```

Comment:
Doesn't it make more sense to do the supported app IDs verification in the VoiceHandler object?

Reply:
sorry for late reply, been really busy, but yes it does i just made this on the quickness, didnt really think about it too much, ill make the changes.